ee/frontend/mobile-replay/transformer/transformers.ts

```diff
@@ -363,44 +401,64 @@ function makeButtonElement(wireframe: wireframeButton, children: serializedNodeW
         : null
 
     return {
-        type: NodeType.Element,
-        tagName: 'button',
-        attributes: inputAttributes(wireframe),
-        id: wireframe.id,
-        childNodes: buttonText ? [{ ...buttonText, id: idSequence.next().value }, ...children] : children,
+        result: {
+            type: NodeType.Element,
+            tagName: 'button',
+            attributes: inputAttributes(wireframe),
+            id: wireframe.id,
+            childNodes: buttonText ? [{ ...buttonText, id: context.idSequence.next().value }, ...children] : children,
+        },
+        context,
     }
 }
 
-function makeSelectOptionElement(option: string, selected: boolean): serializedNodeWithId {
-    const optionId = idSequence.next().value
+function makeSelectOptionElement(
+    option: string,
+    selected: boolean,
+    context: ConversionContext
+): ConversionResult<serializedNodeWithId> {
+    const optionId = context.idSequence.next().value
     return {
-        type: NodeType.Element,
-        tagName: 'option',
-        attributes: {
-            ...(selected ? { selected: selected } : {}),
-            'data-rrweb-id': optionId,
-        },
-        id: optionId,
-        childNodes: [
-            {
-                type: NodeType.Text,
-                textContent: option,
-                id: idSequence.next().value,
+        result: {
+            type: NodeType.Element,
+            tagName: 'option',
+            attributes: {
+                ...(selected ? { selected: selected } : {}),
+                'data-rrweb-id': optionId,
             },
-        ],
+            id: optionId,
+            childNodes: [
+                {
+                    type: NodeType.Text,
+                    textContent: option,
+                    id: context.idSequence.next().value,
+                },
+            ],
+        },
+        context,
     }
 }
 
-function makeSelectElement(wireframe: wireframeSelect, children: serializedNodeWithId[]): serializedNodeWithId | null {
+function makeSelectElement(
+    wireframe: wireframeSelect,
+    children: serializedNodeWithId[],
+    context: ConversionContext
+): ConversionResult<serializedNodeWithId> | null {
     return {
-        type: NodeType.Element,
-        tagName: 'select',
-        attributes: inputAttributes(wireframe),
-        id: wireframe.id,
-        childNodes: [
-            ...(wireframe.options?.map((option) => makeSelectOptionElement(option, wireframe.value === option)) || []),
-            ...children,
-        ],
+        result: {
+            type: NodeType.Element,
+            tagName: 'select',
+            attributes: inputAttributes(wireframe),
+            id: wireframe.id,
+            childNodes: [
+                ...// TODO this won't work once we're editing the context
```

Comment:
Just commenting to make sure you didn't miss this

Reply:
commenting here for the thread

> One thing I don't think I fully understood was the screenshot in the description.

Sorry, yep, the "image" button was below the text area before because the mutations were applying multiple times but not an equal amount of multiple times so the order was changed. I didn't have a screenshot to hand 🙈